### PR TITLE
[appveyor] update file

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -20,5 +25,11 @@ build_script:
...
-  - cmake -G "NMake Makefiles" -DCMAKE_C_COMPILER=clang-cl -DCMAKE_CXX_COMPILER=clang-cl -DCMAKE_Fortran_COMPILER=flang -DCMAKE_BUILD_TYPE=Release ..
-  - nmake VERBOSE=1
+  - ps: |
+      cmake -G "NMake Makefiles" -DCMAKE_C_COMPILER=clang-cl -DCMAKE_CXX_COMPILER=clang-cl -DCMAKE_Fortran_COMPILER=flang -DCMAKE_VERBOSE_MAKEFILE=ON -DCMAKE_BUILD_TYPE=Release ..
+      Push-AppveyorArtifact .\CMakeFiles\CMakeOutput.log
+      Push-AppveyorArtifact .\CMakeFiles\CMakeError.log
+  - ps: |
+      cmake --build . 2>&1 | Out-File build_output.txt
+      Push-AppveyorArtifact .\build_output.txt
+  - nmake VERBOSE=1 check-flang
```
Comment (isuruf) [inline .appveyor.yml]: This is not going to work as the LLVM source is not present

Comment (ghost) [reply .appveyor.yml]: Okay. I saw that on the other PR.

File: .appveyor.yml [hunk 2]
```diff
@@ -20,5 +25,10 @@ build_script:
...
   - set "PATH=%cd%\bin;%PATH%"
   - call "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" amd64
-  - cmake -G "NMake Makefiles" -DCMAKE_C_COMPILER=clang-cl -DCMAKE_CXX_COMPILER=clang-cl -DCMAKE_Fortran_COMPILER=flang -DCMAKE_BUILD_TYPE=Release ..
-  - nmake VERBOSE=1
+  - ps: |
+      cmake -G "NMake Makefiles" -DCMAKE_C_COMPILER=clang-cl -DCMAKE_CXX_COMPILER=clang-cl -DCMAKE_Fortran_COMPILER=flang -DCMAKE_VERBOSE_MAKEFILE=ON -DCMAKE_BUILD_TYPE=Release ..
+      Push-AppveyorArtifact .\CMakeFiles\CMakeOutput.log
+      Push-AppveyorArtifact .\CMakeFiles\CMakeError.log
+  - ps: |
+      cmake --build . 2>&1 | Out-File build_output.txt
```
Comment (isuruf) [inline .appveyor.yml:33]: Doing this will not fail the build

Comment (ghost) [reply .appveyor.yml:33]: That's correct. I thought at the time that the tests would catch it but that's obviously impossible. Sorry for my incompetance.

**Pull request merged**